### ISSUE TITLE
Custom Loss

### ISSUE THREAD

**Issue by redhat12345** [sunziping2016/transfer-tensorflow#2]
Custom Loss

**Comment by sunziping2016** [comment]
Do it yourself.

And pytorch can be a better choice for research if time is important to you.
